src/mbgl/renderer/buckets/symbol_bucket.cpp

```diff
@@ -35,14 +35,13 @@ SymbolBucket::SymbolBucket(Immutable<style::SymbolLayoutProperties::PossiblyEval
       sortUploaded(false),
       justReloaded(false),
       hasVariablePlacement(false),
-      symbolInstances(std::move(symbolInstances_)),
+      symbolInstances(symbolInstances_),
```

Comment:
Could you fix constructor parameters? 
`const std::string bucketName_` => `const std::string& bucketName_` and 
`const std::vector<SymbolInstance>&& symbolInstances_` => `std::vector<SymbolInstance> symbolInstances_`

Reply:
I have a follow-up PR fixing these issues. It is another thing that clang-tidy is complaining.